src/app/pages/Customize.svelte

```diff
@@ -209,12 +259,25 @@
  -->
 <main>
   <div class="customize">
+    <div class="customize__sidebar">
+      {#each projects as project}
+        <div class="customize__sidebar--project" class:selected={project.id === id} on:click={() => id = project.id}>
+          {(project.dictionaryName || "?").substring(0, 1)}
```

Comment:
**tl;dr**: `.substring(0,1)`will break for many languages.

There's no good solution for this, but `.substring(0,1)` is, unfortunately, not good to get the first "letter" of the dictionary name. You want the first [extended grapheme cluster](http://www.unicode.org/reports/tr29/#Grapheme_Cluster_Boundaries) of the name, and there's no easy way to do that in vanilla JavaScript at the moment (aside from [dragging in a dependency](https://www.npmjs.com/package/grapheme-breaker) just for this one task).

Also, because JavaScript chooses a rather annoying way of representing Unicode, `.substring(0, 1)` might actually return a _broken_ character: an  unpaired low surrogate. You can replicate this by getting the first "character" of the 💩  emoji:

```
> "💩".substring(0, 1)
'�'
```

Reply:
I found [this thread](https://stackoverflow.com/questions/46157867/how-to-get-the-nth-unicode-character-from-a-string-in-javascript) on StackOverflow that might be an alternative to `.substring(0,1)`: `[...s][0]` which will give us the first "character" of `s` assuming `s` is not empty.

Reply:
Yes, the ol' [`Array.from(s)[0]` trick](http://eddieantonio.ca/unicode-es6/#/5/4) works here to get the first _code point_. Unfortunately for us, the first code point is not necessarily the first _graphem cluster_, which Matthias Bynens points out in [the accepted answer](https://stackoverflow.com/a/46159939/6626414). 

Try this:

```javascript
> s = ("고추장").normalize("NFD")
'고추장'
> console.log(Array.from(s))[0])
```

Haha, again, this is a very tricky problem to solve in general, for all languages! 😅

Reply:
Ohh that's very interesting... Thanks a lot I'll update the code accordingly!

Reply:
Is there any reason we're going with `NFD` option for unicode normalization?

Reply:
Haha, I would **always** choose NFC. Unless you think "ᄀ" is the correct first letter for "고추장" (I don't know Korean, I just like spicy food).

Reply:
That makes sense! I was wondering about that :) Will make the change accordingly!